doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -192,7 +192,7 @@ nRF5340 Audio
 nRF Machine Learning (Edge Impulse)
 -----------------------------------
 
-|no_changes_yet_note|
+* Bluetooth device name is included in scan response data only if used Bluetooth local identity has no bond.
```

Comment:
```suggestion
* Updated the Bluetooth device name logic.
  The name is now included in scan response data only if the used Bluetooth local identity has no bond.
```
Please propagate to other occurrences.

Reply:
Changed to:
```
* Updated Bluetooth advertising data and scan response data logic.                                   
  The Bluetooth device name is included in scan response data only if used Bluetooth local identity has no bond.
```

Reply:
We actually updated logic related to Bluetooth advertising/scan response data, not logic related to Bluetooth device name itself.

Reply:
Later on we could merge this note with the doc changes introduced here: https://github.com/nrfconnect/sdk-nrf/pull/8457